### ISSUE TITLE
Triunfo rel-1.1.0, Disclaimer "Atualizando..." disappear but feeds remains out-of-date 

### ISSUE THREAD

**Issue by elbidone** [meumobi/RImobi#14]
I've tried to sync feeds of Home but headlines of "Destaques" category remains out-of-date. I need to access to the category to see up-to-date news.

**Comment by t4deu** [comment]
Each home headline make a request to the api/category{id}/items. to get the items of the headline category.

After some tests, I managed to reproduce the error in two different ways:
**Items request fails**
All the RImobi requests fails silently, no message or warning is displayed, if some error occurs then the localstorage cache is used

To fix We can specify what to do when a request fails, We already tried to show alerts every time a request fails, but this has removed because of the amount of alerts displayed.

**items request is slower than the performance request**
This was odd, I observed that when the headline items request is slower than the performance request, the items are saved on the localstorage, but the View isn't updated, probably some problem with the angular digest live cycle.

To fix I need check the "HeadlinesDIrective" for the bug, and possible use $timeout to allow the view be updated
